src/System.Private.CoreLib/shared/System/Lazy.cs

```diff
@@ -499,7 +499,7 @@ internal T ValueForDebugDisplay
         /// </exception>
         /// <remarks>
         /// If <see cref="IsValueCreated"/> is false, accessing <see cref="Value"/> will force initialization.
-        /// Please <see cref="System.Threading.LazyThreadSafetyMode"> for more information on how <see cref="T:System.Threading.Lazy{T}"/> will behave if an exception is thrown
+        /// Please <see cref="System.Threading.LazyThreadSafetyMode"/> for more information on how <see cref="T:System.Threading.Lazy{T}"/> will behave if an exception is thrown
```

Comment:
❔ Can these references be simplified here? I was worried about the possibility of a property named `LazyThreadSafetyMode`, but looking above it seems to be named `Mode` which wouldn't conflict.

```suggestion
        /// Please <see cref="LazyThreadSafetyMode"/> for more information on how <see cref="Lazy{T}"/> will behave if an exception is thrown
```

Reply:
> Can these references be simplified here?

I'm not concerned with cleaning up all of the comments now, only fixing the malformed XML that was causing exceptions in the source.dot.net build.

Reply:
Also, maintaining and polishing XML comments in CoreCLR/CoreFX sources is of limited value today since it is not what the actual documentation is generated from.